draco/core/containers.py

```diff
@@ -781,7 +784,7 @@ class SiderealStream(VisContainer):
         The number of points to divide the RA axis up into.
     """
 
-    _axes = ("ra",)
+    _axes = ("ra", "freq", "stack", "input")
```

Comment:
Why did you change this? They're inherited from VisContainer.

Reply:
I don't know, this doesn't have anything to do with the PR, it must just have slipped in.